src/backend/src/prisma/schema.prisma

```diff
@@ -555,3 +563,67 @@ model Manufacturer {
   dateDeleted   DateTime?
   materials     Material[]
 }
+
+model Calendar {
+  name          String          @id
+  designReviews Design_Review[]
+}
+
+enum Design_Review_Status {
+  UNCONFIRMED
+  CONFIRMED
+  SCHEDULED
+  DONE
+}
+
+enum Design_Review_Team {
+  ELECTRICAL
+  SOFTWARE
+  MECHANICAL
+}
+
+model Design_Review {
+  designReviewId   Int                  @id @default(autoincrement())
+  dateScheduled    DateTime
+  dateCreated      DateTime
+  userCreated      User                 @relation(fields: [userCreatedId], references: [userId], name: "designReviewCreator")
+  userCreatedId    Int
+  status           Design_Review_Status
+  teamType         Design_Review_Team
+  requiredMembers  User[]               @relation(name: "requiredAttendee")
+  optionalMembers  User[]               @relation(name: "optionalAttendee")
+  confirmedMembers User[]               @relation(name: "confirmedAttendee")
+  deniedMembers    User[]               @relation(name: "deniedAttendee")
+  location         String?
+  online           Boolean
+  inPerson         Boolean
+  zoomLink         String?
+  attendees        User[]               @relation(name: "userAttended")
+  dateDeleted      DateTime?
+  userDeleted      User?                @relation(fields: [userDeletedId], references: [userId], name: "designReviewDeleter")
+  userDeletedId    Int?
+  docTemplateLink  String?
+  Calendar         Calendar             @relation(fields: [calendarName], references: [name])
+  calendarName     String
+}
+
+model Schedule_Settings {
+  drScheduleSettingsId Int          @id @default(autoincrement())
+  personalGmail        String
+  personalZoomLink     String
+  availability         Availability @relation(fields: [availabilityId], references: [availabilityId])
+  availabilityId       Int
+  User                 User[]       @relation(name: "settingsUser")
```

Comment:
Should be one to one relation - should be good when schedule settings is optional